_data/simple-icons.json

```
@@ -6949,6 +6949,16 @@
 		"hex": "D50000",
 		"source": "https://www.gsmarena.com"
 	},
+	{
+		"title": "GSoC",
+		"hex": "F9AB00",
+		"source": "https://summerofcode.withgoogle.com",
+		"aliases": {
+			"aka": [
+				"Google Summer of Code"
+			]
+		}
```

```suggestion
		"title": "Google Summer of Code",
		"hex": "F9AB00",
		"source": "https://summerofcode.withgoogle.com",
		"aliases": {
			"aka": [
				"GSoC"
			]
		}
```

The primary name is 'Google Summer of Code' with GSoC being an abbreviation. I would swap the title and AKA.